x-pack/plugins/synthetics/public/legacy_uptime/state/api/has_zip_url_monitors.ts

```diff
@@ -10,5 +10,7 @@ import { SyntheticsHasZipUrlMonitorsResponse } from '../../../../common/types/zi
 import { apiService } from './utils';
 
 export const getHasZipUrlMonitors = async (): Promise<SyntheticsHasZipUrlMonitorsResponse> => {
+  // delay the request to allow the page to render
+  await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
This seems a little hacky; if we must wait for something it'd be better to push this logic into a saga effect?